fred2/sexp_tree.cpp

```diff
@@ -3499,7 +3499,15 @@ int sexp_tree::query_default_argument_available(int op, int i)
 			return 0;
 
 		default:
-			Int3();
+			if (!Dynamic_enums.empty()) {
+				if ((type - First_available_list_id) < (int)Dynamic_enums.size()) {
+					return 1;
+				} else {
+					UNREACHABLE("Unhandled SEXP argument type!");
+				}
+			} else {
+				UNREACHABLE("Unhandled SEXP argument type!");
+			}
```

Comment:
The changes in sexp_tree.cpp and sexp_tree.h should be replicated in the qtfred counterparts of these files.

Reply:
I have no way to build or test qtfred and the code is slightly different. But based on reading it, I think what I did is accurate.

Reply:
A best-effort basis should be good enough for now.